Firmware/MotorControl/encoder.hpp

```diff
@@ -74,16 +74,20 @@ class Encoder : public ODriveIntf::EncoderIntf {
     int32_t shadow_count_ = 0;
     int32_t count_in_cpr_ = 0;
     float interpolation_ = 0.0f;
-    float phase_ = 0.0f;    // [count]
-    float pos_estimate_ = 0.0f;  // [count]
-    float pos_cpr_ = 0.0f;  // [count]
-    float vel_estimate_ = 0.0f;  // [count/s]
+    float phase_ = 0.0f;        // [count]
+    float pos_estimate_counts_ = 0.0f;  // [count]
+    float pos_cpr_counts_ = 0.0f;  // [count]
+    float vel_estimate_counts_ = 0.0f;  // [count/s]
     float pll_kp_ = 0.0f;   // [count/s / count]
     float pll_ki_ = 0.0f;   // [(count/s^2) / count]
     float calib_scan_response_ = 0.0f; // debug report from offset calib
     int32_t pos_abs_ = 0;
     float spi_error_rate_ = 0.0f;
 
+    float pos_estimate_ = 0.0f;  // [rad]
+    float vel_estimate_ = 0.0f;  // [rad]
+    float pos_cpr_ = 0.0f;  // [rad]
```

Comment:
It used to be the case that circular mode was between 0 and `cpr`, because `cpr` was "one turn", i.e. circular mode is within one turn. But now "in one turn" no longer has anything to do with `cpr`.
I would suggest we rename this to `pos_circular_`. Similarly we should rename `setpoints_in_cpr` and other things like that (if any (I forget if there are more)).

Reply:
Fixed in commit a262688